src/pipeline/nodes/VirtualNode.groovy

```diff
@@ -83,7 +83,7 @@ def startFetchLogs(String library_dir, String target_dir){
         ]) {
             dir(target_dir){
                 sh """#!/bin/bash -ex
-                current_dir=`pwd`
+                current_dir=`pwd`"/"
```

Comment:
why do we need this ??!

Reply:
in ansible ```fetch``` if target is ``` asd/qwe``` without ```/``` it will regard it as a file and try to write it.

If the path is a directory and exists already, ansible will throw error "you try to write a file but it's a directory"

so "/" is necessary here.

Reply:
It's running locally.